api.js

```diff
@@ -2,7 +2,7 @@ var mongo = require('mongodb').MongoClient,
     fiddles = null;
 
 module.exports = function(app) {
-    mongo.connect(String(process.env.MONGOHQ_URL), function(err, db) {
+    mongo.connect(process.env.MONGOHQ_URL, function(err, db) {
```

Comment:
Was this causing trouble somewhere? I can't remember why we were casting to a string here, so I am hesitant to remove it.